assets/js/pages/ClusterDetails/HanaClusterDetails.jsx

```diff
@@ -32,6 +32,9 @@ export const enrichNodes = (clusterNodes, hosts) =>
     ...hosts.find(({ hostname }) => hostname === node.name),
   }));
 
+const generateEnrichedSapSystem = (systems) =>
```

Comment:
This new function, and the `enrichedSapSystems`, do they make any sense?
You are just putting id and sid in id and sid.
Why don't you simply pass `sapSystems` to the table?